### PR TITLE
DOCS-4675 add initial skeleton docs

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-4675 add initial skeleton docs

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
modules/ROOT/assets/images/empty.png is here to create the currently empty directories modules/ROOT/assets and modules/ROOT/assets/images .

File: modules/ROOT/pages/planning.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+= Planning
```
Comment (george-hoffer) [inline modules/ROOT/pages/planning.adoc]: Salesforce style issue -- They don't allow gerund headings (ing) -- Maybe "Plan API Community Manager" ?

File: modules/ROOT/pages/setup.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+= Setup
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Installed as Salesforce package.
```
Comment (george-hoffer) [inline modules/ROOT/pages/setup.adoc]: add "a" before Salesforce

File: modules/ROOT/pages/forum-moderation.adoc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+= Forum Moderation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Delete, approve for publication, other features such as frozen users.
```
Comment (george-hoffer) [inline modules/ROOT/pages/forum-moderation.adoc]: Better word choice than "frozen users"

File: modules/ROOT/pages/release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+== 1.0
+
+*May 22, 2019*
+
+=== New Features
```
Comment (george-hoffer) [inline modules/ROOT/pages/release-notes.adoc:12]: You only need headings and content if something changed.

Comment (george-hoffer) [reply modules/ROOT/pages/release-notes.adoc:12]: Please provide a link from the release notes to the user guide and vice versa

**Review by george-hoffer — APPROVED** [review]

File: modules/ROOT/pages/planning.adoc (hunk 1 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/planning.adoc]: Oh, thank you. I'll fix that.

**Pull request merged**